android-client-sdk/src/main/java/com/devcycle/sdk/android/model/Variable.kt

```diff
@@ -66,17 +55,26 @@ class Variable<T> internal constructor(
         required = true,
         description = "Variable value can be a string, number, boolean, or JSON"
     )
+    @SerializedName("value")
     var value: T,
 
     @JsonIgnore
+    @SerializedName("isDefaulted")
     var isDefaulted: Boolean? = null
 ) : PropertyChangeListener {
 
     /**
      * Variable type
      */
     enum class TypeEnum(@get:JsonValue val value: String) {
-        STRING("String"), BOOLEAN("Boolean"), NUMBER("Number"), JSON("JSON");
+        @SerializedName("0")
```

Comment:
whats going on here?

Reply:
lets GSON understand the parsing of the `enum`

Reply:
what's the significance of the serialized name being a specific number? How does that correspond to a type field with value `Boolean` being deserialized to `TypeEnum.Boolean`?

Reply:
Oh, actually.. reading the latest docs.. this isn't required it seems 👀

Reply:
yeah, works without this.. going to remove this annotation :)